src/components/Amalgamation/AmalgamationStatement.vue

```diff
@@ -79,11 +79,18 @@ export default class AmalgamationStatement extends Vue {
     this.courtApproval = this.getAmalgamationCourtApproval
   }
 
-  // Emit a boolean which is whether come with the court approval
+  /**
+   * Emit a value and event to the parent to handle updating.
+   * @param courtApproval The value to emit.
+   */
   @Emit('update')
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   private courtApprovalUpdate (courtApproval: boolean): void {}
 
+  /**
+   * Emit a value and event to the parent to handle validation.
+   * @param valid The value to emit.
+   */
```

Comment:
Also update comment for setCourtApproval ?

Reply:
Just wondering because it's does not have `param` and `return`, should I also update the comment to JSDoc syntax or leave it now?

Reply:
Sure. That way you'll get a nice hover help panel for the function :)